### DISCUSSION TITLE
Enabling Treeview throw exception

### DISCUSSION THREAD

**Discussion by gkielwasser** [directus/directus#8734] [Archive]
Hi,

I'm trying to enable the treeview.

What I have done:
1- create a new collection
2- add a O2M children field + M2O parent opposite field
3- enable Tree View

![Capture](https://user-images.githubusercontent.com/1532438/136994836-c4042bd3-b678-421b-8f54-f8ce2dcb0633.PNG)
When I go on the create page or view page I got the following issue: The children field is broken and not displayed at all.

I have also those 2 errors in the console:
```
vendor.90d6ad72.js:5 ReferenceError: selectionFilters is not defined
vendor.90d6ad72.js:5 TypeError: Cannot read properties of undefined (reading 'collection')
```

Thanks

**Comment by rijkvanzanten** [comment]
Opening a new issue is best 👍🏻  (Make sure to include the exact configuration of this field when you do)

**Reply by gkielwasser** [reply]
I have created [the following issue](https://github.com/directus/directus/issues/8736)
